.azure-pipelines/Linux-CI.yml

```diff
@@ -123,3 +128,10 @@ jobs:
       fi
 
     displayName: Test backend test data
+
+  - script: |
+      if [ '$(documentation)' == '1' ]; then
+        pip install -r docs/docsgen/source/requirements.txt
+        cd docs/docsgen && make html
+      fi
+    displayName: Builds documentation
```

Comment:
So is this builds doc in this CI more like for check/test purpose? If yes, shall we use a more precise displayName?

Reply:
Test documentation?

Reply:
Sounds good!